lib/pxclient.js

```diff
@@ -57,6 +64,25 @@ class PxClient {
         activity.details = details;
         return activity;
     }
+    async fetchRemoteConfig(config) {
+        try {
+            const callData = {
+                url: `https://sapi-${config.px_app_id}.perimeterx.net/config/`,
+                headers: { 'Authorization': `Bearer ${config.px_remote_config_secret}`, 'Accept-Encoding': '' },
+                timeout: 20000,
+            };
+            const res = await p({ url: callData.url, headers: callData.headers, timeout: callData.timeout, method: 'GET' });
```

Comment:
`p` is a low level method, why no using `request.get` ?

Reply:
Because I wanted to use the blocking async/await  and I cant do it to an non async function such request.get